src/krux/printers/cnc.py

```diff
@@ -253,11 +253,15 @@ def print_qr_code(self, qr_code):
             with SDHandler():
                 self.file = open("/sd/qr.nc", "w")
                 super().print_qr_code(qr_code)
-        except:
+        except OSError:
             pass
         finally:
-            self.file.flush()
-            self.file.close()
+            if self.file:
+                self.file.flush()
+                self.file.close()
+
+    def print_string(self, text):
+        """Print a text string. Avoided on CNC"""
```

Comment:
Maybe add a TODO: implement here, or make a github issue saying it needs to be implemented